.github/workflows/build.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches:
       - main
-      - 2022-12-21-fuzz
```

Comment:
unrelated but I assume this was a PR branch and the line was used to test the workflow

Reply:
Maybe there was a time when the fuzz testing was in this repo.